lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -433,6 +435,13 @@ def estimate_diffusion(self, method, max_lag=None):
             is ignored.
             When the method chosen is "ols" an optimal number of lags is estimated as determined by
             [3]_. When the method is set to "gls" all lags are included.
+        localization_variance : float (optional)
+            Estimate of the localization variance. This value can be obtained from estimating an
+            ensemble diffusion constant using `cve`. This parameter is only used when method="cve".
+        localization_variance_variance : float (optional)
+            Estimate of the variance of the localization variance estimate. This value can be
```

Comment:
`Estimate of the variance of the localization variance estimate` 👍

Reply:
:)